support-frontend/assets/pages/contributions-landing/components/StripeCardForm/StripeCardForm.jsx

```diff
@@ -177,6 +162,7 @@ const CardForm = (props: PropTypes) => {
   const [calledRecaptchaRender, setCalledRecaptchaRender] = useState<boolean>(false);
 
   const [zipCode, setZipCode] = useState('');
+  const zipCodeFieldMode: ZipCodeFieldMode = props.country === 'US' ? 'REQUIRED' : 'NONE';
```

Comment:
Or just 
`const zipCodeRequired = props.country === 'US';`

Reply:
Yeah that makes sense. Maybe something like `showZipCodeField`? Just to stress it's either there or not